.prettierrc

```diff
@@ -25,6 +25,10 @@
     {
       "files": [".prettierrc", ".renovaterc.json", "*.json"],
       "options": {"parser": "json"}
+    },
+    {
+      "files": "build-system/compile/bundles.config.extensions.json",
+      "options": {"parser": "json-stringify"}
```

Comment:
nit: While you're here, could we reorder these override blocks? Maybe put the `build-system` rules after the root / wild-card rules?

1. `OWNERS`
2. `*.md`
3. `[".prettierrc", ".renovaterc.json", "*.json"]`
4. `build-system/compile/bundles.config.extensions.json`
5. `build-system/tasks/make-extension/template/**`